tests/torchtune/models/qwen2/test_qwen2_tokenizer.py

```diff
@@ -15,12 +16,13 @@
 
 
 class TestQwen2Tokenizer:
-    def tokenizer(self, template: bool = False):
+    def tokenizer(self, template: bool = False, max_seq_len: Optional[int] = 2048):
         return qwen2_tokenizer(
             path=str(ASSETS / "tiny_bpe_vocab.json"),
             merges_file=str(ASSETS / "tiny_bpe_merges.txt"),
             special_tokens_path=str(ASSETS / "tiny_bpe_tokenizer.json"),
             prompt_template="torchtune.data.ChatMLTemplate" if template else None,
+            max_seq_len=max_seq_len,
```

Comment:
should we keep it null to make sure it doesnt fail when its null?

Reply:
yep